recipes-kernel/linux-firmware/linux-firmware_%.bbappend

```diff
@@ -15,8 +15,8 @@ BRANCH_tibt = "master"
 
 SRC_URI:append = " \
 	git://github.com/varigit/bcm_4343w_fw.git;protocol=https;branch=${BRANCH_brcm};destsuffix=brcm;name=brcm \
-	git://git.ti.com/wilink8-wlan/wl18xx_fw.git;protocol=https;branch=${BRANCH_tiwlan};destsuffix=tiwlan;name=tiwlan \
-	git://git.ti.com/ti-bt/service-packs.git;protocol=https;branch=${BRANCH_tibt};destsuffix=tibt;name=tibt \
+	git://git.ti.com/git/wilink8-wlan/wl18xx_fw.git;protocol=https;branch=${BRANCH_tiwlan};destsuffix=tiwlan;name=tiwlan;depth=1 \
```

Comment:
What does this do? I can't clone that repo: 
$ git clone git://git/git.ti.com/git/wilink8-wlan/wl18xx_fw.git^C
$ man git clone

Reply:
`git://git.ti.com/git/wilink8-wlan/wl18xx_fw.git;protocol=https` would turn into `https://git.ti.com/git/wilink8-wlan/wl18xx_fw.git`.
It looks the content was previously available as `https://git.ti.com/wilink8-wlan/wl18xx_fw.git`, but a `/git/` prefix was added which is causing some issues with the URL.